src/DwitTech.AccountService.Core/Services/ActivationService.cs

```diff
@@ -0,0 +1,63 @@
+﻿using DwitTech.AccountService.Core.Interfaces;
+using DwitTech.AccountService.Core.Utilities;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Mail;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Core.Services
+{
+    public class ActivationService : IActivationService
+    {
+        private static string GetActivationCode()
+        {
+            var activationCode = RandomUtil.GenerateUniqueCode();
+            return activationCode;
+        }
+
+        private string GetNameFromDb(string userEmail) //TODO
```

Comment:
I see no use for this. Why are we getting name from DB? Look at the sequence diagram for the Create account flow and see where this step falls in that flow. Will the name be in the DB then?